ui-admin/src/api/api.tsx

```diff
@@ -349,7 +349,7 @@ export default {
     if (response.ok) {
       return obj
     }
-    return Promise.reject(response)
+    return Promise.reject(obj)
```

Comment:
It's hard to know what impacts this change will have, especially since the value is interpreted by each client of `Api` rather than within `Api` itself. Have you explored all of the usages to make sure that there aren't callers relying on this being `response`?

Reply:
that would be part of the full PR